firmware/Battery.cpp

```diff
@@ -0,0 +1,17 @@
+#include "Battery.h"
+
+Battery::Battery() {}
+
+void Battery::decode(String* strs, HardwareSerial& serial) {
+    serial.println(checkLevel());
+}
+
+float Battery::checkLevel() {
+    int readingPin = A2;
+    float batteryLevel = float(analogRead(A2)) / 1023.0 * 15.0+0.7;
+    return batteryLevel;
```

Comment:
Šta znače ovi brojevi? Mislim da bi bilo jasnije ako ovde piše nešto nalik:
```suggestion
    return float(analogRead(readingPin)) / ADC_RESOLUTION * 3 * ARDUINO_VOLTAGE + MEASUREMENT_OFFSET;
```